pythonfmu/fmi2slave.py

```diff
@@ -250,12 +251,20 @@ def set_string(self, vrs: List[int], values: List[str]):
                     f"Variable with valueReference={vr} is not of type String!"
                 )
 
-    def _get_fmu_state(self) -> Dict[str, Any]:
+    def _get_fmu_state(self) -> Dict[str, any]:
```

Comment:
```suggestion
    def _get_fmu_state(self) -> Dict[str, Any]:
```

This is one of those trick in Python. `Any` is the type (from the `typing` package) documenting a *any* type. But `any` is a standard function of Python to apply on an iterable of Boolean to test if at least one is true (for information `all` function exists too).